nixos/modules/services/networking/dae.nix

```diff
@@ -0,0 +1,42 @@
+{ config, pkgs, lib, ... }:
+
+{
+  options = {
+    services.dae = {
+      enable = lib.options.mkEnableOption (lib.mdDoc "the dae service");
+    };
+  };
+
+  config = lib.mkIf config.services.dae.enable {
+    environment.systemPackages = [ pkgs.dae ];
+
+    networking.firewall.allowedTCPPorts = [ 12345 ];
```

Comment:
This port can't be configured?

Reply:
> This port can't be configured?

Yes, according to dae document: https://dae.v2raya.org/docs/current/FAQ#troubleshoot-firewall

This port is reserved for internal usage, and shall be opened by default.